west.yml

```diff
@@ -72,7 +72,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: pull/2101/head
+      revision: 8411e6a346838a2d0980b28e9add8b29a06d711f
```

Comment:
Bot messed up, this should be done in commit 9265f54e3c63b180048bb04437cd8936c26c2393, please fix (this is a known issue pending resolution).

Reply:
Done